test/extended/cmd.sh

```diff
@@ -143,3 +144,15 @@ os::cmd::expect_success "oc process -f test/extended/fixtures/image-pull-secrets
 os::cmd::try_until_text "oc get pods/my-dc-1-hook-pre -o jsonpath='{.status.containerStatuses[0].imageID}'" "docker"
 os::cmd::expect_success "oc delete all --all"
 os::cmd::expect_success "docker rmi -f ${docker_registry}/image-ns/busybox:latest"
+
+# check to make sure that service serving cert signing works correctly
```

Comment:
> whether its tested
`test/extended/cmd` is not in conformance and therefore not tested on a normal PR job